### PR TITLE
Show sane information about the index ranges on System/Indices

### PR DESCRIPTION
**Pull request by joschi**
Instead of using the "dummy" index range of the current deflector target, the index overview now shows the actual number of messages in the current deflector target and "now" as message date.

### CODE REVIEW

File: app/views/system/indices/index.scala.html [hunk 1]
```diff
@@ -124,19 +125,26 @@
...
                         } else {
-                            Contains messages from @DateHelper.readablePeriodFromNow(index.getRange.getBegin) up to @DateHelper.readablePeriodFromNow(index.getRange.getEnd)
-                            @if(index.getName.equals(deflectorTarget) && index.getInfo != null) {
-                                (@Tools.byteToHuman(index.getInfo.getStoreSizeBytes) /
-                                <span class="number-format" data-format="0,0">@index.getInfo.getDocuments.count</span> messages)
+                            @if(index.getRange.getBegin.isEqual(0L)) {
+                                Contains messages up to @DateHelper.readablePeriodFromNow(index.getRange.getEnd)
+                            } else {
+                                Contains messages from @DateHelper.readablePeriodFromNow(index.getRange.getBegin) up to @DateHelper.readablePeriodFromNow(index.getRange.getEnd)
+                                @if(index.getName.equals(deflectorTarget) && index.getInfo != null) {
```
Comment (edmundoa) [inline app/views/system/indices/index.scala.html]: This branch will never be called (the index will never be a deflector at that point), and is also redundant, the size is also rendered afterwards.

**Comment by edmundoa** [comment]
This also fixes #1127.

File: app/views/system/indices/index.scala.html (hunk 1 above)
Comment (joschi) [inline app/views/system/indices/index.scala.html]: Mild shock

**Comment by edmundoa** [comment]
LGTM :+1:

**Pull request merged**